data/json/items/archery.json

```diff
@@ -400,7 +400,51 @@
       "durability" : 6,
       "clip_size" : 1,
       "reload": 110,
-      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]]
+      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]],
+      "use_action": {
+        "menu_text": "Tighten Limbs",
+        "type": "transform",
+        "target": "compbow_high",
+        "msg": "You tighten the limbs, increasing the strength of the bow."
+      }
+   },
+   {
+      "id" : "compbow_high",
+      "copy-from" : "compbow",
+      "override" : true,
+      "type" : "GUN",
+      "name" : "compound bow (high)",
```

Comment:
Please add `"name_plural"` otherwise we'll get this:

![image](https://user-images.githubusercontent.com/16213433/43266092-2294be58-90f3-11e8-8bd1-99e97ded7e54.png)

Reply:
Is this not intended behaviour?

Reply:
It's better to have it plural to avoid the "(high)s", as "2 compound bows (high)" isn't possible.